.travis.yml

```diff
@@ -1,9 +1,18 @@
-sudo: false
+sudo: true
 services:
   - redis-server
 language: ruby
 rvm:
   - '2.3'
 before_install:
-  - cd ruby
   - gem install bundler -v 1.5.1
+  - cd python
+  - sudo pip install -U pip
+  - sudo pip install setuptools==33.1.1
+  - pip --version
+  - sudo pip install -r requirements.txt
+  - cd ../ruby
+
+after_script:
```

Comment:
My bad. I tried very hard to got both test suites running at once, but couldn't find anything better. If I knew a better way to do this I would have used it.